config/l3afd.cfg

```diff
@@ -68,3 +68,5 @@ cert-dir: /etc/l3af/certs
 cacert-filename: ca.pem
 server-crt-filename: server.crt
 server-key-filename: server.key
+cert-expiry-warning-days: 30
+# how many days before expiry you want warning
```

Comment:
Thanks for at least adding this comment.
FYI, I filed https://github.com/l3af-project/l3afd/issues/79